ReadMe.md

```diff
@@ -1,4 +1,4 @@
-
+Hi, prioritize the work points
```

Comment:
no work now i will assign you

Reply:
index page iruka who right side images pannanum
our serive kojam improve panna pannum with background designs and box shadow podanum

Reply:
okk

Reply:
is the latest updates reflected on host . no styles.css and main.css changes where reflected which i made try to readd files and host it

Reply:
i will start working by tomorrow tell me exactly what i need to

Reply:
ok

Reply:
hosting will happpen on tomorrow without any bugs

Reply:
what about hosting work?

Reply:
give a time like 5-6 or ur suitable time ,on that time lets come online